### PR TITLE
IS-380: replaced nutrition obs data with ic3 nutrition history data

### PR DESCRIPTION
**Pull request by rhenshaw56**

### CODE REVIEW

File: src/layout/SummaryAndForm.jsx [hunk 1]
```diff
@@ -88,6 +88,7 @@ export class SummaryAndForm extends React.Component {
       formSubmittedActionCreator: [
         () => this.props.patient && this.props.patient.uuid && visitActions.fetchPatientActiveVisit(this.props.patient.uuid, this.props.sessionLocation.uuid, ACTIVE_VISITS_REP),
         () => this.props.patient && this.props.patient.uuid && patientActions.getIC3PatientScreeningData(this.props.patient),
+        () => this.props.patient && this.props.patient.uuid && patientActions.getIC3PatientNutritionHistory(this.props.patient),
```
Comment (mogoodrich) [inline src/layout/SummaryAndForm.jsx]: @rhenshaw56  do we need to add this here?  I would think we'd only want/need to fetch the Nutrition History from the ClinicianSummary or NutritionSummary pages?

File: src/screening/ScreeningForm.jsx [hunk 2]
```diff
@@ -16,7 +16,8 @@ class ScreeningForm extends React.Component {
     const formSubmittedActionCreators = [
       () => props.patient && props.patient.uuid && visitActions.fetchPatientActiveVisit(props.patient.uuid,
         this.props.sessionLocation ? this.props.sessionLocation.uuid : null, ACTIVE_VISITS_REP),
-      () => props.patient && props.patient.uuid && patientActions.getIC3PatientScreeningData(props.patient)
+      () => props.patient && props.patient.uuid && patientActions.getIC3PatientScreeningData(props.patient),
+      () => props.patient && props.patient.uuid && patientActions.getIC3PatientNutritionHistory(props.patient),
```
Comment (mogoodrich) [inline src/screening/ScreeningForm.jsx]: Same as my comment from above... can we just do this when the ClinicianSummary or NutritionSummary pages load? Do we need to do it every time any ScreeningForm is loaded?

Comment (rhenshaw56) [reply src/screening/ScreeningForm.jsx]: @mogoodrich so this is for getting the  updated ic3 nutrition history after saving the form, I was hoping that the nutritionSummary and clinicianSummary would both use data from one part of the store

Comment (rhenshaw56) [reply src/screening/ScreeningForm.jsx]: I dropped a reference below

**Pull request merged**